task/script.js

```diff
@@ -220,7 +222,7 @@ taskForm.onsubmit = async (e) => {
   } else {
     delete dataToBeSent.links;
   }
-
+  console.log(dataToBeSent);
```

Comment:
do we need this console log in production?

Reply:
removed